.travis.yml

```diff
@@ -1,25 +1,30 @@
 language: ruby
-dist: precise
+dist: trusty
 cache:
   bundler: true
   directories:
     - node_modules
-sudo: false
+sudo: required
+# https://blog.travis-ci.com/2017-08-29-trusty-image-updates
+group: deprecated-2017Q3
+before_install:
+  - sudo apt-get install qt5-default libqt5webkit5-dev qtdeclarative5-dev gstreamer1.0-plugins-base gstreamer1.0-tools gstreamer1.0-x
+  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
+  - qmake -version
 script:
   - bundle exec bin/setup
   - bundle exec rake travis:ci
-#after_failure:
-#  - cat ./log/test.log
-before_install:
-  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
 env:
   global:
-    - CI_NODE_TOTAL=4
+    - CI_NODE_TOTAL=7
```

Comment:
Curious what bumping this up to 7 does.

Reply:
The build will stall due to unknown reasons. It could be Travis who killed the worker due to system resource issues (memory, disk, etc). Or it might be that we are using very old versions test related gems.

Reply:
Set it to 7 shortens each job but makes the total build time longer since we can run 5 jobs concurrently for free.